src/sandbox_config.ml

```diff
@@ -0,0 +1,129 @@
+open! Stdune
+
+(* ['a t] represents a total map from [Sandbox_mode.t] to ['a] *)
```

Comment:
What about calling this `Sandbox_mode.Dict.t`? This is what we do in other modules such as `Cm_kind`, `Mode`, ...

Reply:
Thanks, that's obviously better!